### PR TITLE
Precommit-action

### PR DESCRIPTION
**Pull request by aboucaud**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
     - id: isort
       args: [--profile, black]
 - repo: https://github.com/asottile/pyupgrade
-  rev: v2.31.0
+  rev: v3.15.1
   hooks:
     - id: pyupgrade
       args: [--py37-plus]
 - repo: https://github.com/PyCQA/flake8
-  rev: 4.0.1
```
Comment (b-biswas) [inline .pre-commit-config.yaml]: test old flake8

**Comment by b-biswas** [comment]
@aboucaud is there a specific reason to add the pre-commit actions? 
In the readme of [pre-commit actions](https://github.com/pre-commit/action) they recommend using [pre-commit ci](https://pre-commit.ci/) instead. 

I added the ci, do we still need actions for something? 

Also, I noticed some errors with the newer versions of flake8, (probably with the args?) so I shifted to the older version that works.

**Comment by aboucaud** [comment]
No specific requirement, no. 

Pre-commit CI did not exist before, which is why I used GH actions.

No need to use both anyway !

**Review by aboucaud — COMMENTED** [review]
LGTM

**Pull request merged**